src/pages/guides/using/asynchronous_calls.md

```diff
@@ -1,6 +1,6 @@
 # Asynchronous Calls - Creating Long Running Actions
 
-When you call a web action or a REST API, the system executes them as synchronous calls (blocking calls) and it waits up to 60 seconds for the action to complete. In many instances, this might just work perfect. However, if your action needs more time to complete the job, than the answer is to use an async call instead of a sync call, as async calls can use up to 30 minutes to complete the work. 
+When you call a web action or a REST API, the system executes them as synchronous calls (blocking calls) and it waits up to 60 seconds for the action to complete. In many instances, this might just work perfect. However, if your action needs more time to complete the job, than the answer is to use an async call instead of a sync call, as async calls can use up to 60 minutes to complete the work. 
```

Comment:
Actually we've updated this to 180 minutes

Reply:
Got it. Just pushed an update to correct this